### PR TITLE
Drop use_epel flag

### PR DESCRIPTION
**Pull request by zmc**
This needs just a bit of testing

### CODE REVIEW

**Comment by zmc** [comment]
Appears to be working fine here:
http://pulpito.front.sepia.ceph.com/zack-2018-05-11_21:00:39-cephmetrics-master-distro-basic-ovh/

Dead job is an infrastructure issue.

File: ansible/roles/cephmetrics-common/tasks/setup_repos.yml [hunk 1]
```diff
@@ -2,37 +2,11 @@
...
-- name: Remove cephmetrics repo
-  file:
-    path: /etc/yum.repos.d/cephmetrics.repo
-    state: absent
-  when:
-    - ansible_pkg_mgr == "yum"
-    - use_epel
-    - devel_mode
-
-- name: Remove old cephmetrics production repo
```
Comment (b-ranto) [inline ansible/roles/cephmetrics-common/tasks/setup_repos.yml]: Shouldn't we keep this at least for production deployments?

Comment (zmc) [reply ansible/roles/cephmetrics-common/tasks/setup_repos.yml]: I don't think anyone that actually deployed RHCS would have had this .repo installed; in fact RHCS would have removed it, even if pre-release deploy code had been run in the past.

Comment (b-ranto) [reply ansible/roles/cephmetrics-common/tasks/setup_repos.yml]: iirc, we deployed this repo before we started shipping with brew and errata. It is probably ok to remove it here but we should add it to the purge.yml.

Comment (b-ranto) [reply ansible/roles/cephmetrics-common/tasks/setup_repos.yml]: I have just checked and purge.yml already removes it so we are ok to merge.

**Pull request merged**